plugin/documents.py

```diff
@@ -837,6 +837,7 @@ def has_capability_async(self, session: Session, capability_path: str) -> bool:
     def purge_changes_async(self) -> None:
         for sv in self.session_views_async():
             sv.purge_changes_async()
+            break
```

Comment:
Added break since technically we only need to purge once per SessionBuffer, regardless how many SessionViews there are.
That said, it was correctly before too because subsequent purges would do nothing.

Reply:
my bad, that was wrong to change as this needs to happen for all sessions in the file